docs/iris/src/whatsnew/1.10.rst

```diff
@@ -0,0 +1,170 @@
+* Support for the NameIII Version 2 file format has been added.
+* The Mercator projection is now supported with defaults for some parameters (_false easting\_, _false northing\_ and _scale factor at projection origin\_).
+
+  * NetCDF files which define a Mercator projection where the _false easting\_, _false northing\_ and _scale factor at projection origin\_ match the defaults will have the projection loaded correctly. Otherwise, a warning will be issued for each parameter that does not match the default and the projection will not be loaded.
+  * NetCDF files which define a steroegraphic projection where the _scale factor at projection origin\_ is equal to 1.0 will have the projection loaded correctly. Otherwise, a warning will be issued and the projection will not be loaded.
+
+* The :mod:`iris.plot` routines :func:`~iris.plot.contour`, :func:`~iris.plot.contourf`, :func:`~iris.plot.outline`, :func:`~iris.plot.pcolor`, :func:`~iris.plot.pcolormesh` and :func:`~iris.plot.points` now support plotting cubes with anonymous dimensions by specifying the *numeric index* of the anonymous dimension within the ``coords`` keyword argument.
+
+  Note that the axis of the anonymous dimension will be plotted in index space.
+* NetCDF loading and saving now supports Cubes that use the LambertConformal coordinate system.
+* The experimental structured Fieldsfile loader :func:`~iris.experimental.fieldsfile.load` has been extended to also load structured PP files.
+
+  Structured loading is a streamlined operation, offering the benefit of a significantly faster loading alternative to the more generic :func:`iris.load` mechanism.
+
+  Note that structured loading is not an optimised wholesale replacement of :func:`iris.load`. Structured loading is restricted to input containing contiguously ordered fields for each phenomenon that repeat regularly over the same vertical levels and times. For further details, see :func:`~iris.experimental.fieldsfile.load` 
+* :mod:`iris.experimental.regrid_conservative` is now compatible with ESMPy v7.
+* Zonal (i.e. longitudinal) mean support has been added to the LBPROC save rule for pp files.
+* Loading of 'little-endian' PP files is now supported.
+* All appropriate :mod:`iris.plot` functions now handle an ``axes`` keyword, allowing use of the object oriented matplotlib interface rather than pyplot.
+* The ability to pass file format object lists into the rules based load pipeline, as used for GRIB, Fields Files and PP has been added.  The :func:`iris.fileformats.pp.load_pairs_from_fields` and :func:`iris.fileformats.grib.load_pairs_from_fields` are provided to produce cubes from such lists.  These lists may have been filtered or altered using the appropriate :mod:`iris.fileformats` modules.
+* Cubes can now have an 'hour' coordinate added with :meth:`iris.coord_categorisation.add_hour`.
+* Time coordinates from PP fields with an lbcode of 31323 are now correctly handled
```

Comment:
I'd like to replace the "correctly handled" with a bit more detail.
Suggest : "Time coordinates from PP fields with an lbcode of the form 3xx23 are now correctly encoded in a 360-day calendar."

@marqh is this a reasonable interpretation ?
This relates to #1893, which is described on the PR in terms of a special case for 31323.  However, the actual code seems more general, as I've tried to indicate.